### DISCUSSION TITLE
想加入更多功能

### DISCUSSION THREAD

**Discussion by yt-theme** [calidion/aex#23] [Ideas]
鉴权 雪花id

**Comment by calidion** [comment]
雪花ID不是web框架功能。

参考这里自行添加：

https://github.com/utkarsh-pro/nodejs-snowflake

鉴权功能中的：
1. 项目相关的，在inject添加即可
2. OAuth相关的，需要用户体系支持，暂时不会在web基本框架内考虑。如果未来开发Web应用框架可以考虑添加到应用框架里。
